### ISSUE TITLE
Dashboard widget error

### ISSUE THREAD

**Issue by ZapaNOR** [verbb/formie#941]
I do not have any widgets, but with the latest release on Craft 4 I get an error whenever I go to the dashboard (every time I log in).

![Screenshot 2022-06-16 at 13 51 10](https://user-images.githubusercontent.com/53942798/174263379-9b6677f8-905f-4ee1-9d55-4b82f5a3ebce.png)

- Plugin version: 2.0.0-beta.13
- Craft version: 4.0.4
- Multi-site: Yes

**Comment by engram-design** [comment]
Strange you get this error if you have no widgets? But regardless, should be fixed for the next release. To get this fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-4 as 2.0.0-beta.13",
  "...": "..."
}
```

Then run `composer update`.

**Comment by ZapaNOR** [comment]
You are of course right, I had one chart widget in the corner, but it was on test with no data so didnt see it.
Can confirm this fixed it.

**Comment by engram-design** [comment]
Fixed in [2.0.0-beta.14](https://github.com/verbb/formie/releases/tag/2.0.0-beta.14)
